packages/smart-wallet/src/offerWatcher.js

```diff
@@ -186,6 +193,17 @@ export const prepareOfferWatcher = baggage => {
               facets.helper.updateStatus({ result: UNPUBLISHED_RESULT });
           }
         },
+        /** @param {Error} err */
+        handleError(err) {
```

Comment:
Maybe add a comment this will likely be called multiple times (aka it needs to handle parallels duplicate calls)

Reply:
I don't get "parallel". Did you mean just "handle duplicate calls?"

Reply:
There will be duplicate call, and since the async operation this initiates is not awaited, they will overlap